internal/merkle/clock/clock.go

```diff
@@ -127,57 +140,96 @@ func (mc *MerkleClock) AddDelta(
 	return link, b, err
 }
 
-func (mc *MerkleClock) checkIfBlockEncryptionEnabled(
+// TODO: rename this to reflect storage
+func (mc *MerkleClock) determineBlockEncryptionData(
```

Comment:
todo: Remove todo and maybe rename to just `determineBlockEncryption`

Reply:
done